Formula/anu.rb

```diff
@@ -1,22 +1,28 @@
+# frozen_string_literal: true
```

Comment:
This is unfortunately going to be removed by GoReleaser, as well as some of the other cosmetic fixes